content/docs/buildpack-author-guide/create-buildpack/build-app.md

```diff
@@ -5,9 +5,20 @@ weight=404
 
 <!-- test:suite=create-buildpack;weight=4 -->
 
-Next, we'll make the build step install dependencies. This will require a few updates to the `build` script.
+Now we'll change the build step you created to install application dependencies. This will require updates to the `build` script such that it performs the following steps:
 
-Let's change `ruby-buildpack/bin/build` to look like the following:
+1. Creates a layer for the Ruby runtime
+1. Downloads the Ruby runtime and installs it to the layer
+1. Installs Bundler (the Ruby dependency manager)
+1. Uses Bundler to install dependencies
+
+By doing this, you'll learn how to create arbitrary layers with your buildpack, and how to read the contents of the app in order to perform actions like downloading dependencies.
+
+Let's begin by changing the `ruby-buildpack/bin/build` so that it creates a layer for Ruby.
+
+### Creating a Layer
+
+A Buildpack layer is represented by a directory inside the [layers directory][layers-dir] provided to our buildpack by the Buildpack execution environment. To create a new layer directory representing the Ruby runtime, change the `build` script to look like this:
 
 <!-- test:file=ruby-buildpack/bin/build -->
```

Comment:
I would think that the tests are failing because the file being created for the test is now incomplete since it's been broken down into multiple sections. [ugo](https://github.com/jromero/ugo#usage), doesn't have support for appending to a file. It would be a great feature to add. :)

Reply:
For now, I could add a file step to the tutorial that says "the full file should look like this"